backend/chainlit/types.py

```diff
@@ -146,16 +149,16 @@ class ChatProfile(DataClassJsonMixin):
 
 
 class FeedbackDict(TypedDict):
-    value: Literal[-1, 0, 1]
-    strategy: FeedbackStrategy
+    forId: str
+    id: Optional[str]
+    value: Literal[0, 1]
     comment: Optional[str]
 
 
 @dataclass
 class Feedback:
     forId: str
-    value: Literal[-1, 0, 1]
-    strategy: FeedbackStrategy = "BINARY"
+    value: Literal[0, 1]
     id: Optional[str] = None
     comment: Optional[str] = None
```

Comment:
@willydouhard why are these changing?

Reply:
Chainlit is only supporting binary feedback so this was never used throughout the project.